server/src/main/java/ru/practicum/shareit/item/ItemServiceImpl.java

```diff
@@ -70,6 +78,9 @@ public ItemDto updateItem(ItemUpdateDto itemUpdateDto) {
         if (itemUpdateDto.getAvailable() != null) {
             item.setAvailable(itemUpdateDto.getAvailable());
         }
+        if (itemUpdateDto.getItemRequestId() != null) {
+            item.setRequest(itemRequestRepository.findById(itemUpdateDto.getItemRequestId()).get());
```

Comment:
```suggestion
            itemRequestRepository.findById(itemUpdateDto.getItemRequestId()).get());
```
Лучше на этом методе сделать обработку исключения, чтобы при отсутствии записи по id была понятная ошибка